chatwoot/chatwoot
# fix: Access token should be hidden/masked by default in super admin dashboard

## Description

Supports masking/unmasking sensitive data such as API Tokens in the super admin dashboard.

## Type of change

Please delete options that are not relevant.


## How Has This Been Tested?

- Click on user avatar, and select `profile settings` option.
- Scroll to bottom of page, and verify that the API token is hidden

## Checklist:
